apps/subscribe-button/src/button/Button.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <button id="podlove-subscribe-button" :style="filling" @click="clickevent()">
+    <icon type="subscribe" :size="iconSize"></icon>
+    <span v-if="format !== 'square'">{{ $t('SUBSCRIBE') }}</span>
+  </button>
+</template>
+
+<script>
+import { mapState } from 'redux-vuex'
+import { selectColor, selectCover, selectFormat, selectSize, selectStyle } from 'store/selectors'
+
+import { Icon } from '@podlove/components'
+
+export default {
+  components: { Icon },
+  props: {
+    clickevent: {
+      type: Function,
+      default: null
+    }
+  },
+  data: mapState({
+    color: selectColor,
+    cover: selectCover,
+    format: selectFormat,
+    size: selectSize,
+    style: selectStyle
+  }),
+  computed: {
+    filling() {
```

Comment:
das würde ich wahrscheinlich in einen selektor berechnen und direct das filling selecten.